app/views/checkouts/index.html.erb

```diff
@@ -2,28 +2,59 @@
 
 <h1>Checkouts</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>User</th>
-      <th>Media object</th>
-      <th>Checkout time</th>
-      <th>Return time</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class="Checkout-index">
+  <% if Checkout.returned_checkouts(current_user.id).count > 0 or current_user.groups.include? 'administrator' %>
```

Comment:
I think this could be tightened up a little.  And I have a tendency to avoid `or` because it has a different precedence than `||` which can get confusing (see https://stackoverflow.com/a/2083118).
```suggestion
  <% if Checkout.returned_checkouts(current_user.id).any? || current_ability.is_administrator? %>
```